app/scripts/components/dashboard/waterpoints.jsx

```diff
@@ -40,13 +43,12 @@ const WaterPoints = React.createClass({
               retry={load}
               state={this.state.waterpointsState} />
         </div>
-        <ChartsContainer>
-          charts for waterpoints...
+        <ChartsContainer
+            onToggle={toggleCharts}
+            state={this.state.layout.charts}>
           <MetricStatus metric="54.65" title="chart.title.functional" total="123456"/>
```

Comment:
I think these should be always visible.
In here we should add new component with chats